saga/seata-saga-annotation/src/main/resources/META-INF/services/org.apache.seata.core.model.ResourceManager

```diff
@@ -14,4 +14,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-io.seata.rm.tcc.resource.parser.TccRegisterResourceParser
\ No newline at end of file
+org.apache.seata.saga.rm.SagaAnnotationResourceManager
```

Comment:
whether the SPI interface in the compatibility package are no longer supported?

Reply:
Yes, we hope to upgrade to the Apache package to use the new features. The compatibility package is not a long solution.

Reply:
RegisterResourceParser  design is too complex, currently only used in tcc proxy registry, and there are few scenarios for extended use, so it is absolutely not compatible

Reply:
ok.